python/utilities/ev_whitelist/create_ev_whitelist_zip.py

```diff
@@ -43,12 +43,15 @@
 class EVCRXHandler(object):
     """Produces updated EV whitelist Chrome extension."""
     def __init__(self, crx_url=CRX_URL):
-        req = requests.get(crx_url)
-        if not req.ok:
+        http = httplib2.Http()
+        http.force_exception_to_status_code = True
+        resp_hdr, resp_body = http.request(crx_url)
```

Comment:
OK.  My opinion is that it's readable enough, and certainly not as terse as Go. :)